bindings/go/descriptor/extensions/scheme.go

```diff
@@ -0,0 +1,24 @@
+package extensions
+
+import "ocm.software/open-component-model/bindings/go/runtime"
+
+var Scheme = runtime.NewScheme()
+
+func init() {
+	MustAddToScheme(Scheme)
+}
+
+func MustAddToScheme(scheme *runtime.Scheme) {
+	obj := &LocalBlob{}
+	scheme.MustRegister(obj, "v1")
+	scheme.MustRegister(obj, "")
```

Comment:
This looks off. We should register it as ungroupedunversioned here probably

Reply:
I will just delete this entire extension, not?

Reply:
I think a LocalBlob is still necessary as an access type no?

Reply:
Doesn't it feel wrong to have that in the descriptor package? 🤔

Reply:
I think thats why it was under extension. Do you have a better place for it?

Reply:
I don't because putting it into another package would be not okay. But extension also feels wrong, because it's such a crucial part. Meh, it's okay. Let's leave it here, if you are fine with the location.